### ISSUE TITLE
Merge draft in word/poetry

### ISSUE THREAD

**Issue by amosonn** [circleofcircles/circleofcircles#5]
I added to poetry an early draft of the idea incorporated. The new version is more precise and broad, but I feel it lost some of the flair of the original. It would be good to merge them somehow.

**Comment by amosonn** [comment]
@simsim314 , do you want to look at this?

**Comment by simsim314** [comment]
Do you mean two versions of poetry?

**Comment by amosonn** [comment]
Yes, inside the file, there's the new version I wrote, and under the title
טיוטה there's an old version of the same idea which I dug up.

**Comment by simsim314** [comment]
לדעתי זה שני ערכים. אחד זה שירה השני רעיון מול מילה.

**Comment by amosonn** [comment]
מה, שתי הפסקאות? הן אומרות כמעט אותו דבר... או שאתה מתכוון שאת הערך כולו
כדאי לפצל?

**Comment by simsim314** [comment]
לדעתי זה ערכים מספיק שונים הגם שדומים. אני בכלל לא יודע אם החלק הראשון הוא נכון ומה המשמעות של קירוב רציונלי לרעיון שאף אחד לא יודע להסביר. מה זה בכלל רעיון שאין אף אחד שמבין.

**Comment by simsim314** [comment]
תן דוגמא לרעיון אחד שנכתבו עליו ספרים ואז הורחבו עליו עוד ספרים.
